source/eitri/docker_wrapper.py

```diff
@@ -1,4 +1,3 @@
-# coding=utf-8
```

Comment:
Are you sure about removing this header?

Reply:
There is no ut8 character in this file so either we use it when we need it or we put it everywhere. Do you concur ?

Reply:
I would rather keep the utf-8 coding definition. 
There might be no utf8 char at the moment, but the entire python codebase is treated as such. What would happened if someone adds a utf8 char in the future ?

Reply:
Ok done